src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -523,6 +523,22 @@ private static void SanityCheckBlob(object obj, string[] blobs)
             }
         }
 
+        private static TargetFrameworkMonikers GetTargetFrameworkMonikers() => PlatformDetection.IsFullFramework ? TargetFrameworkMonikers.NetFramework : TargetFrameworkMonikers.Netcoreapp;
```

Comment:
I think you can just inline this

Reply:
Actually I suggest this is all too elaborate I would just have
```c#
private static int GetBlobIndex()
{
     return PlatformDetection.IsFullFramework ? 1 : 0;
}
```

Reply:
I agree with Dan.   The ternary to get a moniker going into a switch with two cases seems a bit weird.